app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Service Catalog
+subtitle: Track every service across your architecture
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization. This catalog gives you insight into your organization's services and their dependencies. It includes both {{site.base_gateway}} services and external integration resources.
+
+<!-- vale off-->
+{% mermaid %}
+graph LR
+  %% Define styles for icons and nodes
+  classDef resource fill:#E6D0F0
+  classDef konnect fill:#d0e5f3
+  classDef serviceCatalog fill:#e1f7d5
```

Comment:
```suggestion
  classDef konnect stroke-dasharray:3,rx:10,ry:10
  classDef serviceCatalog fill:#d0e5f3,rx:10,ry:10
```
In the interest of consistency across our diagrams, lets use rounded corners and not use purple and green - unless there's a specific purpose to those colours.